debian/control

```diff
@@ -3,8 +3,8 @@ Priority: extra
 Maintainer: Nobody <nobody@kodi.tv>
 Build-Depends: debhelper (>= 9.0.0), cmake,
                kodi-addon-dev, libp8-platform-dev,
-               rapidjson-dev (>= 1.0.2)
```

Comment:
This was correct and there is rapidjson-dev 1.0.2 available in the Kodi Xenial Ubuntu PPAs for building. For newer Ubuntu versions it provided via the distribution repository.

The reason for the build failing for Ubuntu Xenial is the following line, where actually 1.1.0 or higher is required. I don't know which one is the real minimum requirement for the add-on, 1.0.2 or 1.1.0.
https://github.com/flubshi/pvr.waipu/blob/7b93ce857f8f47f4cf7a980060b8fb82d8f6e1cf/FindRapidJSON.cmake#L22

Reply:
Thanks. I am sorry for my stupid trial and error approach. I now locally compiled pvr.waipu with rapidjson 1.0.2. An error message appeared:
`pvr.waipu/src/WaipuData.cpp:412:52: error: ‘class rapidjson::GenericValue<rapidjson::UTF8<> >’ has no member named ‘GetArray’; did you mean ‘SetArray’?`

In [this tutorial](https://rapidjson.org/md_doc_tutorial.html) I found, that this is implemented starting from rapidjson-1.1.0. Thus, the real minimum version required should be 1.1.0. I will adjust the requirement in both files accordingly.